packages/react-components/src/components/xy-output-component.tsx

```diff
@@ -173,7 +180,7 @@ export class XYOutputComponent extends AbstractTreeOutputComponent<AbstractOutpu
                 yAxes: [{ display: false }]
             },
             animation: { duration: 0 },
-            events: [ 'mousedown' ],
+            events: [ 'mousemove', 'mouseout' ],
```

Comment:
if you remove mousemove here, the widget with all the line series are not redrawn for every move (remove also the vertical line from afterChartDraw()), then the tooltip creation is very fast, because the chart is not redrawn, only the tooltip is rendered. 

This could be a compromise for now till we have better performance solution for drawing the vertical line. I'm not sure if I'd prefer to have the vertical line at all, because it's seems confusing to me when doing the time selection (left mouse drag) or right mouse zoom. 

Alternatively, draw the tooltip only on hover after a little delay, like the timeline-chart.

Reply:
Good way to solve the problem. Done.